### ISSUE TITLE
bug: Refuses to patch on tablet (a13)

### ISSUE THREAD

**Issue by Dajova** [ReVanced/revanced-manager#1357]
### Bug description

I've tried to patch both youtube revanced and youtube music revanced on both my tablet and phone, but it refuses to work recently on my tablet, while it works fine on my phone. Said provided log triggers everytime when I try to patch either of the apps (manually or via patcher).
It was working fine a couple of months ago, when I last updated the apps.
My phone is a older a9 version, while my tablet is newer on a13.

### Version of ReVanced Manager and version & name of application you tried to patch

1.11.2

### Installation type

Non-root

### Device logs

```shell
~ Device Info
ReVanced Manager: 1.11.2
Build: release
Model: SM-T220
Android version: 13
Supported architectures: arm64-v8a, armeabi-v7a, armeabi

~ Patch Info
App: com.google.android.youtube v18.38.44
Patches version: v2.193.0
Patches: Hide ads, Video ads, Copy video url, External downloads, Disable fine scrubbing gesture, Seekbar tapping, Swipe controls, Disable auto captions, Premium heading, Hide video action buttons, Hide autoplay button, Hide captions button, Hide cast button, Navigation buttons, Hide player buttons, Hide album cards, Hide breaking news shelf, Comments, Hide crowdfunding box, Hide endscreen cards, Hide filter bar, Hide floating microphone button, Hide layout components, Hide info cards, Hide 'Load more' button, Hide email address, Player flyout menu, Hide seekbar, Hide Shorts components, Hide timestamp, Hide watermark, Disable player popup panels, Custom player overlay opacity, Return YouTube Dislike, Wide searchbar, SponsorBlock, Spoof app version, Disable Shorts on startup, Enable tablet layout, Tablet mini player, Theme, Alternative thumbnails, Always autorepeat, Enable debugging, Client spoof, Bypass URL redirects, Open links externally, Vanced MicroG support, Minimized playback, Disable zoom haptics, HDR auto brightness, Remember video quality, Playback speed, Old video quality menu

~ Settings
Enabled changing patches: false
Enabled universal patches: true
Enabled experimental patches: true
Patches source: revanced/revanced-patches
Integration source: revanced/revanced-integrations

~ Logs
Copying APK
Reading APK
Decoding app manifest
Loading patches
Merging integrations
Deleting existing resource cache directory
Decoding resources
An error occurred:
brut.androlib.exceptions.AndrolibException: Could not generate: values-sk/plurals.xml
	at brut.androlib.res.ResourcesDecoder.generateValuesFile(ResourcesDecoder.java:223)
	at brut.androlib.res.ResourcesDecoder.decodeResources(ResourcesDecoder.java:182)
	at app.revanced.patcher.data.ResourceContext.decodeResources$revanced_patcher(ResourceContext.kt:50)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:194)
	at app.revanced.patcher.Patcher$apply$1.invoke(Unknown Source:8)
	at app.revanced.patcher.Patcher$apply$1.invoke(Unknown Source:4)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$2$1.invokeSuspend(MainActivity.kt:256)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$23(MainActivity.kt:255)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$iczzv34SoBa_J1fRmhQ1dE2ClMs(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda3.run(Unknown Source:22)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: brut.directory.DirectoryException: file must be a directory: /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-JNMTBM/cache/res/values-sk
	at brut.directory.FileDirectory.<init>(FileDirectory.java:38)
	at brut.directory.FileDirectory.createDirLocal(FileDirectory.java:64)
	at brut.directory.AbstractDirectory.createDir(AbstractDirectory.java:149)
	at brut.directory.AbstractDirectory.getFileOutput(AbstractDirectory.java:122)
	at brut.androlib.res.ResourcesDecoder.generateValuesFile(ResourcesDecoder.java:205)
	... 20 more
```

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by oSumAtrIX** [comment]
Caused by: brut.directory.DirectoryException: file must be a directory: /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-JNMTBM/cache/res/values-sk

Relevant line ^

**Comment by oSumAtrIX** [comment]
Do you have enough space?

**Comment by ILoveOpenSourceApplications** [comment]
This issue, as of today, is nearly 4 months old and no follow-up is done by the OP. Can be closed as there's no other reports of similar kind.

**Comment by ILoveOpenSourceApplications** [comment]
Also @oSumAtrIX, can you remove the `waiting on author` tags from closed issues here and over at `revanced patches`.

**Comment by oSumAtrIX** [comment]
There's no need to, you can filter for multiple tags

**Comment by KobeW50** [comment]
This issue is still relevant. A user on discord had this exact patching error

https://pastebin.com/bNqZh5gQ
